app/server.go

```diff
@@ -885,7 +885,7 @@ func (s *Server) initRudder(endpoint string) {
 			config.Verbose = true
 			config.BatchSize = 1
 		}
-		client, err := rudder.NewWithConfig(RUDDER_KEY, config)
+		client, err := rudder.NewWithConfig(RUDDER_KEY, RUDDER_DATAPLANE_URL, config)
```

Comment:
Since we're passing `RUDDER_DATAPLANE_URL` directly now, is the `config.Endpoint` setting still relevant?

Reply:
I'll leave it to @grundleborg to take a call on that. I'd like to keep this PR _just_ about updating dependencies with minimal disruption in the codebase.